assets/pages/weekly-subscription-landing/components/weeklyCta.scss

```diff
@@ -21,12 +23,13 @@
 
 .weekly-cta__content,
 .weekly-cta svg {
-  display: inline-block;
-  vertical-align: middle;
+  flex: 0 0 auto;
+  display: block;
 }
 
 .weekly-cta svg {
   height: $gu-v-spacing * 2;
+  width: 2em;
```

Comment:
Don't we usually express in `px` and let the compiler convert to `em`? I may be wrong.